features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/services/login/introspect.jag

```diff
@@ -33,31 +34,21 @@
 
     var data = {token: token }
     var introspectResult = post(introspectEndpoint, data , {
-        "Authorization": "Basic YWRtaW46YWRtaW4=", // TODO: get super admin username password from configs and do encoding ~tmkb
+        "Authorization": "Basic " + utils.getBase64EncodedAdminCredentials(),
         "Content-Type": "application/x-www-form-urlencoded"
     });
-
+    introspectResult.data = JSON.parse(introspectResult.data);
```

Comment:
use post() fourth param as "json" instead for using JSON.parse and reassigning

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/6667/commits/60ed8577f196f7059f542c18c0b5001939c01ad3